andela/travel_tool_front
# #160663195 Fix the Approve or Reject Button Text

#### What does this PR do?
fix approve or reject button text

#### Description of Task to be completed?
- Change the approval button text when clicked to Approved
- Change the reject button text when clicked to Rejected

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login into the application
- Navigate to the Approval page
- Click on any requests
- Click on Approve or Reject
- The text should change accordingly "Approved or Rejected"

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160663195](https://www.pivotaltracker.com/story/show/160663195)


#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-21 at 6 13 56 am" src="https://user-images.githubusercontent.com/28366272/45861626-8ecdd200-bd65-11e8-8fa7-fc31268784d8.png">

#### Questions:
N/A
